### ISSUE TITLE
Code for creating subject is duplicated

### ISSUE THREAD

**Issue by antoinefalisse** [stanfordnmbl/opencap-viewer#294]
We have more or less the same code [here ](https://github.com/stanfordnmbl/opencap-viewer/blob/main/src/components/pages/Step4.vue)and [here](https://github.com/stanfordnmbl/opencap-viewer/blob/main/src/components/pages/Subjects.vue). Would be nice to have it only once. @AlbertoCasasOrtiz is that something you could do? Thanks.

**Comment by AlbertoCasasOrtiz** [comment]
Sure! I just added to tomorrow sprint.
